spec/controllers/application_controller_spec.rb

```diff
@@ -0,0 +1,163 @@
+require 'spec_helper'
+
+describe ApplicationController do
+
+
+  it 'loads the homepage' do 
+    get '/'
+    expect(last_response.status).to eq(200)
+    expect(last_response.body).to include("Welcome to Fwitter")
+  end
+
+  it 'loads the signup page' do 
+    get '/signup'
```

Comment:
It feels like there should be a few describe blocks here around the different paths